webapp-django/manage.py

```diff
@@ -8,11 +8,15 @@
 import sys
 
 
-if __name__ == '__main__':
+def main(argv):
     # Note! If you for some reason change that change the wsgi
     # starting-point script too.
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'crashstats.settings')
 
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line(sys.argv)
+    execute_from_command_line(argv)
+
+
+if __name__ == '__main__':
+    main(sys.argv)
```

Comment:
This change just adjusts this file so it's possible to run `main` with different arg sets.